src/index.html

```diff
@@ -4,9 +4,368 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          class="header__logo-img"
+          src="./images/logo.png"
+          alt="Moyo header__logo"
+        >
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a href="index.html" class="header__link is-active">Apple</a>
+          </li>
+          <li class="header__item">
```

Comment:
add empty line between siblings
```suggestion
          </li>
          
          <li class="header__item">
```